src/timelines/preamble.js

```diff
@@ -1,17 +1,50 @@
-import experimentStart from '../trials/experimentStart'
-import startCode from '../trials/startCode'
-import userId from '../trials/userId'
-import holdUpMarker from '../trials/holdUpMarker'
-import { AT_HOME, IS_ELECTRON } from '../config/main'
-
-console.log('at_home', AT_HOME)
-console.log('env at home', process.env.REACT_APP_AT_HOME)
+import { jsPsych } from "jspsych-react";
+import { eventCodes, lang } from "../config/main";
+import holdUpMarker from "../trials/holdUpMarker";
+import { showMessage, userId } from "@brown-ccv/behavioral-task-trials";
+import { config, AT_HOME } from "../config/main";
+console.log("at_home", AT_HOME);
+console.log("env at home", process.env.REACT_APP_AT_HOME);
 const preamble = {
-  type: 'html_keyboard_response',
-  stimulus: '',
-  timeline: (AT_HOME || !IS_ELECTRON) ?
-    [experimentStart(), userId()] :
-    [experimentStart(), userId(), holdUpMarker(), startCode()]
-}
+  type: "html_keyboard_response",
+  stimulus: "",
+  timeline: !config.USE_PHOTODIODE
+    ? [
+        showMessage(config, {
+          responseType: "html_button_response",
+          message: `<h1>${lang.task.name}</h1>`,
+          responseEndsTrial: true,
+          buttons: [lang.prompt.continue.button],
+        }),
+        userId(jsPsych, config, {
+          responseType: "html_keyboard_response",
```

Comment:
shouldn't `html_keyboard_response` be the default for `userId`? Also, sometimes `message` is a whole stimulus, and sometimes it seems to be just a text string - could you make sure those are consistent on the library side? I'd vote for something like `message` (and similar variants) are just strings, whereas `stimulus` (and variants` are html, and if just message is passed, it's shoved into `h1` tags, but if stimulus is passed, it's ignored - thougths?

Reply:
I can definitely revisit this piece of code and use a better way of passing the message